pkg/mount/mount_linux.go

```diff
@@ -151,7 +151,7 @@ type Pool struct {
 // determine whether it has already been mounted.
 func (p *Pool) Mount(mounter Mounter, flags uintptr) (*MountPoint, error) {
 	for _, m := range p.MountPoints {
-		if m.Device == mounter.DevName() {
+		if m.Device == filepath.Join("/dev", mounter.DevName()) {
```

Comment:
I'm not sure this is right, given that below `path := filepath.Join(p.tmpDir, mounter.DevName())`

Reply:
I print them out and it's correct, for example:
m.Device: /dev/nvme0n1p1, mounter.DevName(): nvme0n1p2
So m.Device is the complete device path but mounter.DevName() is only the device name, so w/o this fix the if comparison will never be true and keep mounting the devices that have already been mounted.